packages/@react-stately/virtualizer/src/useVirtualizerState.ts

```diff
@@ -80,22 +80,37 @@ export function useVirtualizerState<T extends object, V, W>(opts: VirtualizerPro
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  return {
+  let setVisibleRect = useCallback((rect) => {
+    virtualizer.visibleRect = rect;
+  }, [virtualizer]);
+  let startScrolling = useCallback(() => {
+    virtualizer.startScrolling();
+    setScrolling(true);
+  }, [virtualizer]);
+  let endScrolling = useCallback(() => {
+    virtualizer.endScrolling();
+    setScrolling(false);
+  }, [virtualizer]);
+
+  let state = useMemo(() => ({
```

Comment:
memo state so it doesn't cause unnecessary onLoadMore calls, this appeared after blurring the combobox and it beginning to exit in 16 and 17